packages/types/src/Metadata/v0/Modules.ts

```diff
@@ -124,7 +124,7 @@ export class ModuleMetadata extends Struct {
 
 export class StorageFunctionModifier extends Enum {
   constructor (value?: any) {
-    super(['Optional', 'Default', 'Required'], value);
+    super(['Optional', 'fallback', 'Required'], value);
```

Comment:
This is a backward compatibility breaking change, which doesn't matter anymore because no one should be using v0 now.

Reply:
Ah, nice catch! This is a find/replace gone wrong, I'll fix it.

Reply:
> which doesn't matter anymore because no one should be using v0 now

It actually matters, because v1-v4 all use v0's StorageFunctionModifier